src/main/java/by/onliner/constants/RecorderConstants.java

```diff
@@ -0,0 +1,8 @@
+package by.onliner.constants;
+
+public class RecorderConstants {
```

Comment:
Do you need a separate class for that or it can be stored in the class where it's used

Reply:
Moved to the recorder class